egs/iam/s5/local/unk_arc_post_to_transcription.py

```diff
@@ -1,14 +1,35 @@
 #!/usr/bin/env python
 
+""" This module will be used by scripts for open vocabulary setup.
+ If the hypothesis transcription contains <unk>, then it will replace 
+ the <unk> with the word predicted by <unk> model. It is currently 
+ supported only for triphone setup.
+
+ Args:
+  phones: File name of a file that contains the phones.txt, (symbol-table for phones).
+          phone and phoneID, Eg. a 217, phoneID of 'a' is 217. 
+  words: File name of a file that contains the words.txt, (symbol-table for words). 
+         word and wordID. Eg. ACCOUNTANCY 234, wordID of 'ACCOUNTANCY' is 234.
+  unk: ID of <unk>. Eg. 231
+  input-ark: file containing hypothesis transcription with <unk>
+  out-ark: file containing hypothesis transcription without <unk>
+  
+  Eg. local/unk_arc_post_to_transcription.py lang/phones.txt lang/words.txt data/lang/oov.int
+
+  Eg. hypothesis transcription: A move to <unk> mr. gaitskell.
+      converted transcription: A move to stop mr. gaitskell.
```

Comment:
The example of 1best-arc-post is wrong. You should probably give an example like
en_4156-A_030763-031116 4 11 1 no n_B o_E
and explain the format (also explain in the help info): <utt-id> <arc-start-frame> <arc-duration> <arc-post> <word> <pronunciaiton>